web_src/less/_base.less

```diff
@@ -1224,6 +1225,7 @@ table th[data-sortt-desc] {
 
 .emoji,
 .reaction {
+    font-family: @emoji-fonts;
```

Comment:
```suggestion
```

I think it's unneccesary and would probably break if we ever render normal text inside these selectors.

Reply:
True, `<img>` alt texts are very real.